### DISCUSSION TITLE
[v4.1] Set default Migrations directory

### DISCUSSION THREAD

**Discussion by maidiries** [adonisjs/core#2468] [Help] [answered]
Can I set the default directory for **Migrations** files in v4.1?

I know that is solved in 5.0 version with .adonisrc.json, but we have a project 4.1 and we need move Migrations files outside root Adonis project folder.
It is posible?

**Comment by artem-a** [comment, marked as answer]
Hi @maidiries  you can try to use hooks

**hooks.js**
```javascript
'use strict'

const path = require('path')
const { hooks } = require('@adonisjs/ignitor')

hooks.before.aceCommand(async () => {
  const Helpers = use('Helpers')

  Helpers.migrationsPath = function (toFile = '') {
    return path.join('/some/new/path/to/migrations', toFile)
  }
})
```

it should help))

**Reply by equeraltowenea** [reply]
Works correctly! Thanks!
